websites/management/commands/update_departments.py

```diff
@@ -0,0 +1,33 @@
+from functools import reduce
+
+from django.core.management import BaseCommand
+from django.db import transaction
+from django.db.models import Q
+
+from websites.models import WebsiteContent
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        filter_set = ["21W", "CMS"]
+        filter_query_set = [
+            "website__name__istartswith",
+            "website__short_id__istartswith",
+        ]
+
+        query_set = WebsiteContent.objects.filter(
+            reduce(
+                lambda x, y: x | y,
+                [Q(**{key: value}) for key, value in zip(filter_query_set, filter_set)],
+            ),
+        )
+
+        query_set = query_set.filter(type="sitemetadata")
+        with transaction.atomic():
+            for sitemetadata in query_set.iterator():
+                if not sitemetadata.metadata["department_numbers"]:
```

Comment:
Is it certain that only sites with no department numbers at all need this change?  For instance, site [CMS.930](https://ocw-studio.odl.mit.edu/sites/cms-930-media-education-and-the-marketplace-fall-2001/type/metadata/) already has department number 21G.

Reply:
I think this PR is good to go as long as it's okay to wipe out any other existing department numbers for these courses and replace with `CMS-W`.  @pdpinch any thoughts on this?

Reply:
We need to preserve any existing department assignments. 

One point of clarification: courses can and should be in multiple departments. I may open another issue to do another data migration to fix up other missing departments.

Reply:
@pdpinch @mbertrand  if there are more issues with departments then we should probably convert this back to a django command and I can also refactor it so  we could use it to fix other departments as well.

Reply:
Sure, that would be fine, it it takes a command line parameter like `--department <my_department>` or `--departments <my_dept_1>, <my_dept_2>`